client/src/components/Comment.vue

```diff
@@ -87,6 +87,8 @@ const props = defineProps({
   },
 });
 
+const commentWithLinks = computed(() => sanitizeComment(props.comment.message));
```

Comment:
Wenn man den Kommentar wieder bearbeiten möchte, liegen alle Zeichen in encodeter Form dar. Diese müssten dann wieder decoded werden.

Reply:
Das wird in dem sanitize PR verhindert. Das hier war nur um XSS im live preview zu verhindern.